automation/components-functests.setup.sh

```diff
@@ -34,14 +34,14 @@ make cluster-down cluster-up
 
 # Export .kubeconfig full path, so it will be possible
```

Comment:
KUBECONFIG is not needed since you switched from kubectl to cluster/kubectl.sh, is it?

Reply:
This is actually needed, some tests simply need KUBECONFIG (for example [knmstate](https://github.com/nmstate/kubernetes-nmstate/blob/master/hack/run-e2e-test-handler.sh#L4) and [ovs-cni](https://github.com/kubevirt/ovs-cni/blob/master/hack/functests.sh#L23))

Reply:
But it is not a general requirement for all the e2e lanes. Would it make sense to have this export in each component-specific script? Keep it close to the user and don't turn this into a junkyard of all that may be needed.

Reply:
I agree, but keep in mind this script only runs on tier1 test scripts. 
it's a general requirement for tier1 tests, since they all run on external cluster (CNAO's)

Reply:
Well, I don't have a terribly strong argument against it. Let's go with your proposal then.